.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "eslint-config-keep",
+    "plugin:react/recommended"
```

Comment:
Any chance we can flip these? I'd like to maintain consistency on the Keep side for the rules where `eslint-config-keep` sets one and `react/recommended` a different one.

Reply:
(Doing this would probably best be coordinated with a rebase to redo the commit that applies fixes, since the thing that tipped me off was the addition of trailing commas for formatting.)

Reply:
I flipped these and re-ran `lint:fix` and nothing changed. 🤔 Investigating.

Reply:
I just realized whatever order they're in, the choice on commas that we made in eslint-config-keep is to use the eslint default. So we don't force it to anything. Which means no matter what, the fact that `plugin:react/recommended` forces it will override the default.

Given the fact that `eslint-config-keep` basically decides not to decide, I'm comfortable with having the frontend stuff differ in departments where `plugin:react/recommended` decides to decide lol.